ramls/allowed-service-points.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": ["array", "null"],
+  "description": "List of allowed pickup service points",
+  "default": null,
+  "minItems": 1,
+  "uniqueItems": true,
+  "items": {
+    "type": "object",
+    "javaType": "org.folio.circulation.domain.AllowedServicePoint",
```

Comment:
Does it make any difference if we specify this java-type here or not? We are not generating any code from this schema anyway...

Reply:
done